### PR TITLE
[wasm] provisioning: run sanitize.py with python3

### PR DESCRIPTION
**Pull request by lambdageek**

### CODE REVIEW

**Review by radical — APPROVED** [review]
LGTM, assuming the wasm jobs pass!

File: src/mono/wasm/sanitize.py [hunk 1]
```diff
@@ -21,7 +22,7 @@ def remove(*paths):
 
 
 def rewrite_package_json(path):
-    package = open(path, "rb+")
+    package = open(path, "r+")
```
Comment (lewing) [inline src/mono/wasm/sanitize.py:25]: did you see an issue with this? I saw something without it

Comment (lewing) [reply src/mono/wasm/sanitize.py:25]: I see the behavior changed in 3

**Pull request merged**